src/jvm/main/org/jetbrains/kotlinx/lincheck/trace/TracePoint.kt

```diff
@@ -334,6 +335,19 @@ internal class MethodCallTracePoint(
     fun initializeOwnerName(ownerName: String) {
         this.ownerName = ownerName
     }
+    
+    fun initializeActorResult(result: Result?) {
+        check(isActor)
+        this.returnedValue = when (result) {
+            null -> ReturnedValueResult.VoidResult
+            is ExceptionResult -> when (result.throwable) {
+                is LincheckAnalysisAbortedError -> ReturnedValueResult.ActorHungResult
```

Comment:
Is it always true that `LincheckAnalysisAbortedError` is thrown only in case of a hung? I think in general, it can also mean other things --- the actor could have been aborted for various reasons.

Reply:
If that is the case we do not test for it. For what I understand from the code base threads can be aborted on live/deadlocks or when there is an internal error. In the last case I think we only show the stacktrace of the internal error and not the execution trace.

Could you provide an example where LincheckAbortedError is the result of one of the actors in the trace and the execution was not dead/live locekd?

I am open to suggestions if you think there are better ways of detecting hung actors

Reply:
> Could you provide an example where LincheckAbortedError is the result of one of the actors in the trace and the execution was not dead/live locked?

I do not have concrete example, just uneasy feeling. However, I agree, current architecture of actors & aborting does not allow us easily implement a better approach, so let's leave it as is for now.